docs/index.md

```diff
@@ -70,6 +70,22 @@ Use Office Scripts in Excel on the web to automate your common tasks. Explore th
 <h2>Other resources</h2>
 <p>Use the following resources to learn about the APIs that Office Scripts use to interact with workbooks in Excel on the web, ask questions about Office Scripts in Excel on the web, or request features for Office Scripts in Excel on the web.</p>
 <ul class="panelContent cardsF cols cols4" style="display:flex!important;">
+    <li>
+        <div class="cardSize">
+            <div class="cardPadding">
+                <div class="card">
+                    <div class="cardImageOuter">
+                        <div class="cardImage">
+                            <a href="https://support.office.com/article/office-scripts-settings-in-m365-19d3c51a-6ca2-40ab-978d-60fa49554dcf" target="_blank"><img src="images/index-landing-page/i_upgrade.svg" alt="Enable Office Scripts" /></a>
+                        </div>
+                    </div>
+                    <div class="cardText">
+                        <a href="https://support.office.com/article/office-scripts-settings-in-m365-19d3c51a-6ca2-40ab-978d-60fa49554dcf" target="_blank"><h3>Enable Office Scripts</h3></a>
```

Comment:
FYI - not sure who owns the page, but there's a typo (missing space) in step 1: 

https://support.office.com/en-us/article/office-scripts-settings-in-m365-19d3c51a-6ca2-40ab-978d-60fa49554dcf

![image](https://user-images.githubusercontent.com/3019529/71927159-b9ee5780-3149-11ea-800f-6752c7bf8b05.png)

Reply:
Good catch. I edited the file, which should be integrated soon.

FWIW, while MARVEL owns support.office.com content, I think everyone in our org can edit those files.